06_puppy/alextmz/main_test.go

```diff
@@ -0,0 +1,20 @@
+package main
+
+import (
+	"bytes"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestMain(t *testing.T) {
+	want := "Read Puppy by ID: {1 Dogo white 50}\n"
+
+	t.Run("main test", func(t *testing.T) {
```

Comment:
no need for t.Run if there is only a single testcase. unless i'm missing something.

Reply:
Agree. 
I guess I just like to see the tree in a hierarchical  way (Testfunction/file, Test function, individual tests inside the function) but I agree that there's no point to it beside cosmetics which aren't worth the price of a harder to read code, thus should be taken off. Will watch this trait as I feel I'm probably doing this frequently.

before:
```
=== RUN   TestMain/main_test
--- PASS: TestMain (0.00s)
    --- PASS: TestMain/main_test (0.00s)
```
changed to:
```
=== RUN   TestMain
--- PASS: TestMain (0.00s)
```